tests/console_test.go

```diff
@@ -88,7 +89,8 @@ var _ = Describe("[rfe_id:127][posneg:negative][crit:medium][vendor:cnv-qe@redha
 			Context("with a cirros image", func() {
 
 				It("[test_id:1588]should return that we are running cirros", func() {
-					vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
+					vmi := libvmi.NewCirros()
+					vmi.ObjectMeta.Namespace = util.NamespaceTestDefault
```

Comment:
This should not be done. On creation of the entity/VMI the namespace is provided, and not as part of the spec.
Ref: [here](https://github.com/kubevirt/kubevirt/blob/5654c762bd5f62ebebfe0fd1b5e45508c0986832/tests/console_test.go#L58).

Reply:
Oh. Oddly, without it I was getting
```
            <*errors.errorString | 0xc003540170>: {
                s: "an empty namespace may not be set when a resource name is provided",
            }
            an empty namespace may not be set when a resource name is provided
        occurred^[[0m

        tests/utils.go:2886
```